tests/test_forward_selection.py

```diff
@@ -4,21 +4,97 @@
 from sklearn.datasets import make_friedman1
 from sklearn.linear_model import LinearRegression
 
+from feature_selection import forward_selection
+
+def scorer(X, y):
+     """
+     Sample custom scorer that fits a model and returns
+     an appropriate score for the feature selection problem.
+     Parameters
+     ----------
+     X : array-like of shape (n_samples, n_features)
+         Test samples
+     y : array-like of shape (n_samples,) or (n_samples, n_outputs)
+         True values for X, used for training
+     Returns
+     -------
+     Error of scorer
+     """
+     lr = LinearRegression().fit(X, y)
+     return 1 - lr.score(X, y)
+
+
 def test_forward_selection():    
     """
     This test creates a dataset that has 5 features that are are used to compute `y`.
     The remaining 5 features are independent of `y`.
     This test should select the first 5 feature columns used to compute `y` more than the second set of 5 independent features.
     """
-    data, target = make_friedman1(n_samples=200, n_features=10, random_state=10)
-    features = forward_selection(scorer, data, target, min_features = 4, max_features=5)
-    features = np.sort(features)
-    assert (features is np.array([0, 1, 3, 4]) or features is np.array([0, 1, 2, 3, 4]))
 
-    # Test min/max number of features working correctly
-    features = forward_selection(scorer, data, target, min_features = 3, max_features=8)
-    assert len(features) > 2
-    assert len(features) < 9
+    data, target = make_friedman1(n_samples=200, n_features=10, random_state=10)
+    results = forward_selection(scorer, data, target, min_features = 4, max_features=5)
```

Comment:
There needs to be a test to prove that this function is working. For the Friedman1 data set features 0, 1, 3, 4 should be identified. This assert is necessary. Please include include. The reason for the sort is to make sure the order is the same because sometimes the selector picks different orders.
```suggestion
    # Test min/max number of features working correctly
    results = forward_selection(scorer, data, target, min_features = 4, max_features=4)
    results = np.sort(results)
    assert (results is np.array([0, 1, 3, 4]))
```